package.json

```diff
@@ -41,7 +41,8 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --only_4xx --ignore-status-codes 403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
+    "proof:all": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
```

Comment:
I do feel like the `proof:all` script could be a little underbaked here. I expect it will need to evolve as we further develop our manual scan process.

 I expect that the manual process will be less about total remediation and more about just noticing when things are getting moved, confirming the content we are linking to is still the same, etc.  So for at least a little while I think it is best to understand where all of our failures are and fine tune what we want to exclude when we aren't in the pressure situation of unblocking merges. 

tl;dr: I lean towards keeping the `proof:all` script broad _for now_ so that we can capture all failures for awhile, and then have the option to narrow the scan once we have a better understanding of what the project needs.

Reply:
As long as `npm proof` will catch _actual_ failures then I am ok with moving forward as is! 👍